projects/igniteui-angular/src/lib/core/utils.ts

```diff
@@ -18,6 +18,12 @@ export function cloneArray(array, deep?: boolean) {
 export function cloneObject(object: any) {
     return JSON.parse(JSON.stringify(object));
 }
+/**
+ *@hidden
+ */
+export function isObjectEmpty(object: any) {
```

Comment:
This is only used to check if a row transaction has any changes, right?
Checking `Object.keys({}).length` should be enough, the transaction API guarantees it's an object already.
Plus checking constructor really isn't very generic as it won't work with instances that extend Object for example.